third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -18,32 +18,26 @@
  ****************************************************************/
 package org.apache.james.linshare;
 
-import static org.apache.james.linshare.LinshareFixture.USER_1;
-import static org.apache.james.linshare.LinshareFixture.USER_CREDENTIAL_MAP;
-import static org.apache.james.linshare.client.LinshareAPI.Headers.ACCEPT_APPLICATION_JSON;
-
-import java.util.List;
-import java.util.Optional;
-
-import org.apache.james.linshare.client.Document;
-import org.apache.james.linshare.client.LinshareAPI;
-import org.apache.james.linshare.client.User;
-import org.apache.james.utils.FakeSmtp;
-import org.junit.jupiter.api.extension.BeforeEachCallback;
-import org.junit.jupiter.api.extension.ExtensionContext;
-
 import com.github.fge.lambdas.Throwing;
-
-import feign.Feign;
-import feign.Headers;
-import feign.Logger;
-import feign.Param;
-import feign.RequestLine;
+import com.google.common.annotations.VisibleForTesting;
+import feign.*;
```

Comment:
We avoid wildcard imports so that people knows exactly what is imported (avoid confusions)